x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/install_elastic_agent.tsx

```diff
@@ -328,16 +328,77 @@ export function InstallElasticAgent() {
                 <>
                   <EuiText color="subdued">
                     <p>
-                      {i18n.translate(
-                        'xpack.observability_onboarding.installElasticAgent.installStep.description',
-                        {
-                          defaultMessage:
-                            'Select a platform and run the command to install in your Terminal, enroll, and start the Elastic Agent. Do this for each host. For other platforms, see our downloads page.  Review host requirements and other installation options.',
-                        }
-                      )}
+                      <FormattedMessage
+                        id="xpack.observability_onboarding.installElasticAgent.installStep.description"
+                        defaultMessage="Select your platform, and run the install command in your terminal to enroll and start the Elastic Agent. Do this for each host. Review {hostRequirementsLink} before installing."
+                        values={{
+                          hostRequirementsLink: (
+                            <EuiLink
+                              external
+                              href="https://www.elastic.co/guide/en/fleet/8.7/elastic-agent-installation.html"
+                            >
+                              host requirements and other installation options
```

Comment:
this should still be wrapped with i18n.translate, even inside the FormattedMessage

Reply:
Good catch!

Reply:
Addressed in [d53b1b5](https://github.com/elastic/kibana/pull/162148/commits/d53b1b5be52cc65841e2deb3d0ac6dbef1eda755)